packages/elements/src/number-field/index.ts

```diff
@@ -169,6 +179,26 @@ export class NumberField extends FormFieldElement {
   @query('[part=spinner-down]')
   private spinnerDownEl?: HTMLInputElement;
 
+  /**
+   * An object's returned from setTimeout to use with repeat delay.
+   */
+  private timerRepeatDelay: NodeJS.Timeout | undefined = undefined;
+
+  /**
+   * An object's returned from setInterval to use with repeat rate.
+   */
+  private timerRepeatRate: NodeJS.Timeout | undefined = undefined;
+
+  /**
+   * Called after the component is first rendered
+   * @param changedProperties Properties which have changed
+   * @returns {void}
+   */
+  protected override firstUpdated(changedProperties: PropertyValues): void {
+    super.firstUpdated(changedProperties);
+    document.addEventListener('tapend', this.resetTimer);
```

Comment:
I feel uncomfortable with `document.addEventListener` how about `this.addEventListener` does it have any concern?

Reply:
Using `this.addEventListener` does `tapend` isn't fired. This causes number-field does infinite count up when long press and drag out of number-field.

Reply:
Oh I see, it necessary to detect tapend from anywhere inside DOM document.
Does will it cause any concerns in further? for example
1. when number-field in side shadow DOM
2. when have to many number field elements

Reply:
1. Tested on color-dialog and time-picker. It works well if they had spinner.
2. Added condition to not call clearTimer at every tapend.